pystan/tests/test_rstan_stanfit.py

```diff
@@ -21,8 +21,8 @@ def test_init_zero_exception_inf_grad(self):
         """
         sm = StanModel(model_code=code)
         assertRaisesRegex = self.assertRaisesRegexp if PY2 else self.assertRaisesRegex
-        with assertRaisesRegex(RuntimeError, 'Gradient evaluated at the initial value is not finite.'):
-            sm.sampling(init='0', iter=1)
+        with assertRaisesRegex(RuntimeError, ''):
```

Comment:
does the message get to the user somehow? What's the new exception message? (If there's an exception message, that could be checked here.)